example/satosa/pyeudiw_dynamic_backend.yaml

```diff
@@ -0,0 +1,221 @@
+module: pyeudiw.satosa.dynamic_backend.DynamicBackend
+name: OpenID4VP
+
+config:
+
+  dynamic_backend:
```

Comment:
dyanmic_backed doesn't means nothing. According to what we have said during our alignment meeting, each class MUST be configured within the corresponding endpoint, in the section where the endpoints are defined

````
  endpoints:
      pre_request: 
        class: pyeudiw.satosa.default.request_handler
        module: OpenID4VPBackend
        path: '/pre-request'
      ...
````

and so on. 
Please remove "Default" from all the names.

Reply:
the base class is the one of the entire backend
https://github.com/italia/eudi-wallet-it-python/pull/229/commits/1e3c9fb685bad6f9fbb51680f40f132a21b21e40#diff-cd382c85588277c666bbd8c9e5c32a931650eab9e0f44188a52a5c2eae28b2f2R1